bestiary/core/models.py

```diff
@@ -149,3 +175,40 @@ class Meta:
 
     def __str__(self):
         return '%s - %s - %s - %s - %s' % (self.ouid, self.trx, self.op_type, self.entity_type, self.target)
+
+
+class Ecosystem(EntityBase):
+    """Model class for Ecosystem objects.
+
+    This class is meant to represent an abstract set of projects
+    which may share a common context.
+
+    Every ecosystem object must have a unique identifier `id` and a
+    name (`name`). Optionally, it may have a title (`title`) and a
+    `description` field containing a brief text describing what
+    the ecosystem is about.
+
+    :param id: Identifier of the ecosystem
+    :param name: Name of the ecosystem
+    :param title: Title of the ecosystem
+    :param description: Description of the ecosystem
+    """
+    id = AutoField(primary_key=True,
+                   help_text='Ecosystem identifier')
+    name = CharField(unique=True,
+                     max_length=MAX_SIZE_CHAR_FIELD,
```

Comment:
I defined a new variable for the `name` and `title` size called `MAX_SIZE_SHORT_CHAR_FIELD` which value is 80 because I saw it is a common value for this type of fields. Does this make sense to you?

Reply:
Yes, that makes sense.